storage/innobase/row/row0mysql.cc

```diff
@@ -2176,6 +2177,12 @@ static dberr_t row_insert_for_mysql_using_ins_graph(const byte *mysql_rec,
 @param[in,out]  prebuilt        prebuilt struct in MySQL handle
 @return error code or DB_SUCCESS*/
 dberr_t row_insert_for_mysql(const byte *mysql_rec, row_prebuilt_t *prebuilt) {
+  raii::Sentry<> compress_heap_cleaner([prebuilt] {
```

Comment:
Can you please put a comment on why you are freeing compress_heap after insert?

Reply:
Here you go.... :)